_targets.R

```diff
@@ -99,6 +99,9 @@ NLCD_folders <- c(
 ## https://www.sciencebase.gov/catalog/item/57e2ac2fe4b0908250045981
 NADP_sb_id <- '57e2ac2fe4b0908250045981'
 
+## FORESCE list of FORESCE years
+FORESCE_years <- c('1960', '1970', '1980', '1990', '2000')
```

Comment:
Adding 1940 and 1950 to prepare for eventually predicting the 1960 historic drought conditions
```suggestion
FORESCE_years <- c('1940', '1950', '1960', '1970', '1980', '1990', '2000')
```